language/id.yml

```diff
@@ -1,63 +1,311 @@
+  **Larangan Baru Federasi**
+  **Federasi: **{}
+  **Administrator Federasi: **{}
+  **Pengguna: ** {}
+  **ID Pengguna: ** {}
+  **Alasan lama**: {}
+  **Alasan baru**: {}
+fed-ban-info: |
+  **Larangan Baru Federasi**
+  **Federasi: **{}
+  **Administrator Federasi: **{}
+  **Pengguna: ** {}
+  **ID Pengguna: ** {}
+  **Alasan**: {}
+fed-user-not-banned: Pengguna ini tidak diblokir dalam federeasi ini!
+fed-unban-info: |
+  **Membatalkan Larangan Federasi**
+  **Federasi: **{}
+  **Administrator Federasi: **{}
+  **Pengguna: **{}
+  **ID Pengguna: **`{}`
+fed-stat-banned: |
+ Pengguna ini di blokir dalam federasi ini.
+  **Alasan: ** {}
+  **Diblokir dalam federasi: ** {}
+  
+  
+fed-stat-not-banned: Pengguna ini tidak di blokir dalam federasi itu!
+fed-stat-multi: Pengguna ini sudah di blokir dalam federasi ini.\n
+fed-stat-multi-info: " -  **{}**(`{}`)\n    **Alasan: **{}\n"
+fed-stat-multi-not-banned: Pengguna ini tidak di blokir dalam federasi manapun!
+fed-backup-empty: Tidak ada data federasi.
+fed-restore-done: Selesai memulihkan file cadangan.
+fed-myfeds-owner: "Kamu adalah **pemilik** dari federasi ini:\n"
+fed-myfeds-admin: "\nKamu adalah **Administrator** dalam federasi ini:\n"
+fed-myfeds-no-admin: Sepertinya anda tidak menjadi Administrator di federasi manapun.
+fed-set-log-args: "Untuk membuat log federasi di saluran atau grup, silahkan gunakan federasi ID yang jelas dan benar untuk dihubungkan.\neg: `/setfedlog federasi ID`"
+fed-not-found: Federasi tidak ditemukan!
+fed-check-identity: Tolong tanyakan kepada pemilik federasi untuk konfirmasi tindakan ini dengan mengklik tombol dibawah ini.
+fed-log-set-group: "Grup ini diatur sebagai **{name}** catatan"
+fed-log-set-chnl: "Saluran ini telah ditetapkan sebagai catatan federasi untuk **{}**\nall tindakan terkait federasi akan dicatat di sini."
+fed-invalid-identity: Pengguna yang mengklik tombol konfirmasi bukanlah pemilik federasi.\nHanya pemilik federasi yang dapat menetapkan catatan federasi.
+fed-log-unset: Catatan federasi dari **{}** telah dibatalkan.
+fed-autoban: "Pengguna {} dilarang dalam federasi ini (**{}**), dan sudah dikeluarkan.\n**Reason:** {}\n**Banned on:** {}"
+#endregion
+
 #region language
 language-button: Bahasa
 language-help: |
   Bot ini hadir dengan dukungan multi bahasa.
-  Anda dapat memilih preferensi bahasa Anda!\n
+  Anda dapat memilih preferensi bahasa anda!\n
   ** Bahasa yang tersedia: **
   - 🇺🇸 Bahasa Inggris.
-  - 🇮🇩 Indonesia.\n
+  - 🇮🇩 Bahasa Indonesia.\n
   Perintah:
-  × /setlang | /lang | /language : setel bahasa yang anda inginkan.\n
+  × /setlang | /lang | /language : Mengganti bahasa yang saya gunakan.\n
   **Catatan**
-  Anda dapat menggunakan perintah ini pada grup untuk mengubah pengaturan bahasa pada grup itu.\n
+  Anda dapat menggunakan perintah ini pada grup anda untuk mengganti bahasa yang digunakan.\n
   Bantu kami mencakup lebih banyak bahasa dengan berkontribusi di [Crowdin!] (https://crowdin.com/project/anjani-bot)
```

Comment:
I think we can delete this `Bantu kami mencakup lebih banyak bahasa dengan berkontribusi di [Crowdin!] (https://crowdin.com/project/anjani-bot)` since we no longger use crowdin.
please change delete this too at [en.yml](https://github.com/userbotindo/Anjani/blob/47476389272634e91e9248902410f7b6aa3b45a7/language/en.yml#L234) 😁

Reply:
sure